src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -3,6 +3,7 @@ Microsoft.CodeAnalysis.AdditionalDocument
 Microsoft.CodeAnalysis.AnalyzerConfigDocument
 Microsoft.CodeAnalysis.ApplyChangesKind.AddAnalyzerConfigDocument = 17 -> Microsoft.CodeAnalysis.ApplyChangesKind
 Microsoft.CodeAnalysis.ApplyChangesKind.ChangeAnalyzerConfigDocument = 19 -> Microsoft.CodeAnalysis.ApplyChangesKind
+Microsoft.CodeAnalysis.ApplyChangesKind.ChangeDefaultNamespace = 20 -> Microsoft.CodeAnalysis.ApplyChangesKind
```

Comment:
I'm wondering if we should just make this a ChangeProjectInfo -- we already have one for ChangeDocumentInfo, and I don't want an enum entry every time we have to make a new project property that gets updated.

Reply:
thanks for a quick response - that's actually a very good idea. I renamed it accordingly